reference/customers.v3.yml

```diff
@@ -3918,6 +3918,14 @@ components:
           description: It determines if the customer is signed up to receive either product review or abandoned cart emails or recieve both emails.
         store_credit_amounts:
           $ref: '#/components/schemas/CustomerStoredCreditAmounts'
+        origin_channel_id:
+          description: Channel ID the customer has been created format
```

Comment:
Unclear on what this definition means. Please rephrase for clarity

Reply:
@traci707 I think this should be `Channel ID the customer has been created from`